setup.py

```diff
@@ -32,6 +33,12 @@
     print("Borg requires Python %d.%d or later" % min_python)
     sys.exit(1)
 
+try:
+    __import__("Cython")
```

Comment:
just "import Cython" does not work here?

Reply:
i guess that would create a unused variable

Reply:
my thinking was that if Cython is not always available it's better to keep it out of the namespace to encourage only using it in parts of the code where it is definitely available.

Reply:
i've used stuff like that in the past:
```
try:
    import X
except ImportError:
    X = None
```
So, you always have X and it is used as a flag at the same time, so you can just do:
```
if X:
    ...
```

Reply:
That looks better. I've updated the code to use that.